.github/workflows/release-node-bindings.yml

```diff
@@ -245,9 +247,7 @@ jobs:
       - name: Enable corepack
         run: corepack enable
       - name: Publish to NPM
-        run: |
-          cd bindings_node
-          npm publish
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        uses: JS-DevTools/npm-publish@v3
+        with:
+          token: ${{ secrets.NPM_TOKEN }}
```

Comment:
Don't you also need to pass in the GITHUB_TOKEN secret as well?

Reply:
shouldn't need that token as this action doesn't seem to interact with the GitHub API. at least according to the action docs, they don't mention needing it.

Reply:
Then I'm not sure what adding the id-token permission does above.

Although now that I've looked at the docs for the action, you're right that it probably won't matter until you set the `provenance` flag to true.

Reply:
hmm, in that case, i'll add the token back. the publish config does enable provenance.

Reply:
Based on their docs I have no idea how you're supposed to configure use both the Github token and a NPM token, since there's only one field for either. But putting it into the env (like you just did) feels like a logical guess. Otherwise you might have to dig through the code.